### PR TITLE
[WIP] add seq mode to awg customised instrument

### PR DESCRIPTION
**Pull request by nataliejpg**
TODO: 
- [x] TEST

### CODE REVIEW

**Comment by nataliejpg** [comment]
@jenshnielsen tested

File: qdev_wrappers/customised_instruments/AWG5014_ext.py [hunk 1]
```diff
@@ -4,12 +4,39 @@
...
+            for i in range(self.sequence_length()):
+                self.set_sqel_loopcnt_to_inf(i)
+
+    def _get_seq_mode(self):
+        if self.get_sqel_loopcnt() == '1':
+            return False
+        else:
+            return True
+
+    def send_and_load_awg_file(self, awg_file, filename):
```
Comment (jenshnielsen) [inline qdev_wrappers/customised_instruments/AWG5014_ext.py]: This function looks general enough that it perhaps belongs in the driver?

Comment (nataliejpg) [reply qdev_wrappers/customised_instruments/AWG5014_ext.py]: Possibly. I think up to taste to be honest, 'make' is already in there as are 'send' and 'load' and a few combinations. Maybe @WilliamHPNielsen has an opinion?

**Comment by nataliejpg** [comment]
No longer necessary as we expect to use the awg with the parameteric sequencer and so this is overruled by the repest_mode parameter on that

**Pull request closed without merging**